src/app/model/area.ts

```diff
@@ -6,7 +6,12 @@ export class Area {
     parentId: Number | null;
 }
 
-export class Type {
+export interface AutoSuggested {
```

Comment:
`AutoSuggested` is now unused, as we only use strings in `AutocompleteBarComponent`